modules/adformBidAdapter.js

```diff
@@ -37,7 +37,7 @@ function AdformAdapter() {
 
     request.unshift('//' + globalParams[0][1] + '/adx/?rp=4');
 
-    request.push('auctionId=' + encodeURIComponent(params.bidderRequestId));
+    request.push('auctionId=' + params.requestId);
```

Comment:
why was the `encodeURIComponent` dropped?

Reply:
I believe it is not necessary, because requestId is UUID.